apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/main/java/gov/cms/model/dsl/codegen/plugin/GenerateSqlFromDslMojo.java

```diff
@@ -218,10 +213,6 @@ private List<ColumnBean> getPrimaryKeyColumns(RootBean root, MappingBean mapping
   private List<ColumnBean> getAllColumns(RootBean root, MappingBean mapping) {
     // this map ensures no column is included twice
     final var columns = new LinkedHashMap<String, ColumnBean>();
-    for (JoinBean join : mapping.getNonArrayJoins()) {
-      final var joinColumn = mapping.mapJoinToParentColumnBean(root, join);
-      joinColumn.ifPresent(c -> columns.put(c.getColumnName(), c));
-    }
     for (ColumnBean column : mapping.getTable().getColumns()) {
       columns.put(column.getColumnName(), column);
     }
```

Comment:
`root` doesn't seem to be used in this method anymore

Reply:
Thanks.  I have fixed these and removed some other unused methods.